bluemira/base/builder.py

```diff
@@ -73,20 +111,20 @@ def __init__(
         self.designer = designer
 
     @abc.abstractproperty
-    def param_cls(self) -> Union[Type[_ParameterFrameT], None]:
+    def param_cls(self) -> Union[Type[ParameterFrame], None]:
         """The class to hold this builder's parameters."""
         pass
 
     @abc.abstractmethod
-    def build(self) -> _ComponentManagerT:
-        """Build the component."""
-        return Component(self.name)
+    def build(self) -> ComponentManager:
+        """Build the component and return a component manager instance."""
+        pass
```

Comment:
I've done this as it's probably a bit confusing that this returned a `Component` not a `ComponentManager`. We now have the `component_tree` helper to generate this for us anyway